compiler/ic/packed_ast.nim

```diff
@@ -139,7 +139,9 @@ proc addModuleId*(tree: var PackedTree; s: ModuleId; info: PackedLineInfo) =
 proc addSymDef*(tree: var PackedTree; s: SymId; info: PackedLineInfo) =
   tree.nodes.add PackedNode(kind: nkSym, operand: int32(s), info: info)
 
-proc isAtom*(tree: PackedTree; pos: int): bool {.inline.} = tree.nodes[pos].kind <= nkNilLit
+proc isAtom*(tree: PackedTree; pos: int): bool {.inline.} =
+  let kind = tree.nodes[pos].kind
+  kind <= nkNilLit or kind == nkCommentStmt
```

Comment:
It's probably a bug in the making of we don't make this a comparison with `nkWithoutSons`

Reply:
Yeah I think so too, I'm making a seperate PR that does a sweep across the codebase that replaces these kinds of checks with the respective sets so I figured I'd leave this one as minimal as can be